docs/credentials.md

```
@@ -0,0 +1,121 @@
+`git` と `docker login` はどちらもデフォルトでは認証情報をディスクに平文（または base64
+エンコードしているだけの実質平文）で残す。fnox で管理しているのはアプリ・API 用の secret で、
+git/Docker 自身の認証情報はそれぞれのツールが用意している credential helper 機構に任せる方が
+自然なため、fnox とは別に `chezmoi` 側でこの2つを管理している。
+
+## git: `credential.helper`
+
+HTTPS 経由で git を使うときに毎回 ID/パスワードを聞かれないよう、認証情報を OS の keychain や
+メモリキャッシュに保存する仕組み。`dot_config/git/config.tmpl` の `[credential]` ブロックで
+OS ごとに設定を分けている:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)